src/utils/format.js

```diff
@@ -96,16 +96,6 @@ export function formatDecodeByteArray(bytesArray) {
   return String.fromCharCode(...bytesArray)
 }
 
-export function formatNameStatus(name) {
```

Comment:
I removed this logic as there is no way to determine whether the name is in auction or not. It was not even possible before.